lib/tasks/doc_facility.rake

```diff
@@ -0,0 +1,56 @@
+require 'open-uri'
+require 'csv'
+
+namespace :doc do
+  desc 'fill out doc_facility table and link records to status and profile'
+  task :doc_facilities do 
+    csv_path = 'lib/Prison.Facilities-Table.1.csv'
+    csv = []
+    facil=Doc::Status.distinct.pluck(:facility)
+    facil_prof=Doc::Profile.distinct.pluck(:facility)
+    status_profile_facil = facil + facil_prof
+    status_profile_facil = status_profile_facil.uniq
+
+    
+
+
+    CSV.foreach('lib/Prison.Facilities-Table.1.csv') do |row|
+        
+        next if row[0] == 'FACILITY'
```

Comment:
```suggestion
    CSV.foreach('lib/Prison.Facilities-Table.1.csv', headers: true)) do |row|
```
^^ if you add headers:true it should tell it to skip the first line so you don't need to do `next if row[0] == 'FACILITY'`